### PR TITLE
Updates before workshop

### PR DESCRIPTION
**Pull request by catherine-martlin**
These are just a series of small updates to have the output numbers from queries and file size checks match what currently shows up when run in the notebook.

### CODE REVIEW

**Comment by catherine-martlin** [comment]
Looks like the Actions workflow needs an update - it's looking for R at a `master` branch, but I think they changes the default to `v2-branch`: https://github.com/r-lib/actions

File: .github/workflows/website.yml [hunk 1]
```diff
@@ -48,7 +48,7 @@ jobs:
 
       - name: Set up R
         if: steps.check-rmd.outputs.count != 0
-        uses: r-lib/actions/setup-r@master
+        uses: r-lib/actions/setup-r@v2-branch
```
Comment (abostroem) [inline .github/workflows/website.yml:51]: was this breaking anything? I agree it's good to keep in sync with main carpentries regardless
@ErinBecker can you confirm this change?

**Comment by abostroem** [comment]
@catherine-martlin thank you so much for this PR (and your previous ones)! I can definitely get it merged in the next few weeks. Please let me know if you have an upcoming workshop on a shorter time scale and I can raise the priority of working on it.

**Comment by catherine-martlin** [comment]
@abostroem - thank you! We are teaching a class at ST this Wednesday/Thursday (November 9th/10th), but I totally understand how short this notice is so now worries if that's too quick a turnover. I can just take notes on those small changes as they were more updates to query results and didn't affect running any of the code. (I found them while going through all the sections and as of last week everything was running fully!)

File: .github/workflows/website.yml (hunk 1 above)
Comment (catherine-martlin) [reply .github/workflows/website.yml:51]: This didn't break any of the notebooks/class code, but it did break the CI checks below which was why I updated it. It's totally fine to move this change to a different branch, etc, if there is a better workflow or need to separate it from the other changes.

**Comment by abostroem** [comment]
These all look good. Apologies for not getting them merged before your workshop. merging now. Thanks again @catherine-martlin

**Pull request merged**